ansible/cleanup/roles/version/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Ok it would be best to squash all these commits because I don't think the github browser based review system is showing me your latest commits.  I still see this in the files section.